app/tasks/upload_with_rclone.py

```
@@ -0,0 +1,169 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)